sql/sql_class.cc

```diff
@@ -1353,7 +1353,6 @@ void THD::cleanup(void) {
   if (wsrep_cs().state() != wsrep::client_state::s_none) {
```

Comment:
Shouldn't the client state and wsrep_client_thread be handled the same way?

With the client state, it's set to s_none here, and later it's reset to s_idle specifically in sql_parse for these commands, which seems to be a hack.